fbchat/client.py

```diff
@@ -797,18 +799,26 @@ def fetchUnread(self):
         :raises: FBchatException if request failed
```

Comment:
`fetchUnread` seems to work now, but it's still not documented, perhaps you could write something in there?

Reply:
Done